ui/components/multichain/address-copy-button/index.scss

```diff
@@ -8,7 +8,7 @@
 
 .custody-logo {
   height: 20px;
-  background-color: var(--brand-colors-white-white000);
+  background-color: var(--brand-colors-white);
   padding: 1px;
```

Comment:
### After
Background of custody logo is white

<img width="577" alt="Screenshot 2024-06-10 at 3 13 42 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/c9867f4c-cca3-452c-9fea-18e595a343c5">
<img width="530" alt="Screenshot 2024-06-10 at 3 13 46 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/58a674f1-0f8b-475a-abc8-202a80d71291">